backend/src/main/java/WebApi.java

```diff
@@ -1,15 +1,82 @@
+import api.customer.CustomerController;
+import api.MaterialInterface;
+import api.PrintingInterface;
+import api.customer.CustomerAPI;
+import com.google.gson.Gson;
+import model.Customer;
+import model.Order;
+import model.OrderedPart;
+
 import static spark.Spark.*;
 
 /**
  * Starts a restapi att localhost:4567
  */
 public class WebApi {
-    private static WebInterface webInterface = new WebInterfaceImpl();
+    private static CustomerAPI ci = new CustomerController();
+    private static MaterialInterface mi;
+    private static PrintingInterface pi;
+
+    private static Gson gson = new Gson();
 
     public static void main(String[] args) {
+        long start = System.currentTimeMillis();
+        System.out.println("STARTED ENDPIONT SETUP");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)